shared/idmap/idmapset_linux.go

```diff
@@ -641,22 +641,27 @@ func (set *IdmapSet) doUidshiftIntoContainer(dir string, testmode bool, how stri
 			// Dump capabilities
 			if fi.Mode()&os.ModeSymlink == 0 {
 				caps, err = GetCaps(path)
-				if err != nil {
-					return err
+				switch err {
+				case syscall.EOPNOTSUPP:
+					logger.Warnf("llistxattr is not supported on %s", path)
+				case nil:
+					// noop
+				default:
+					return errors.Wrapf(err, "failed getting caps on %s: %v", path, err)
```

Comment:
Please can you change this to "Failed..." to be consistent with other LXD error messages. Thanks